opentelemetry/src/test/java/io/quarkus/ts/opentelemetry/OpentelemetryIT.java

```diff
@@ -0,0 +1,84 @@
+package io.quarkus.ts.opentelemetry;
+
+import static io.restassured.RestAssured.given;
+import static org.awaitility.Awaitility.await;
+import static org.hamcrest.Matchers.containsInAnyOrder;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.text.IsEqualIgnoringCase.equalToIgnoringCase;
+
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.http.HttpStatus;
+import org.hamcrest.Matcher;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.DisabledOnOs;
+import org.junit.jupiter.api.condition.OS;
+
+import io.quarkus.test.bootstrap.JaegerService;
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.JaegerContainer;
+import io.quarkus.test.services.QuarkusApplication;
+import io.restassured.response.Response;
+
+@QuarkusScenario
+@DisabledOnOs(value = OS.WINDOWS, disabledReason = "Windows does not support Linux Containers / Testcontainers")
+public class OpentelemetryIT {
+    private static final int GRPC_COLLECTOR_PORT = 14250;
+
+    private Response resp;
+
+    @JaegerContainer(image = "jaegertracing/all-in-one:latest", restPort = GRPC_COLLECTOR_PORT, expectedLog = "\"Health Check state change\",\"status\":\"ready\"")
```

Comment:
`GRPC_COLLECTOR_PORT` -> is using the default port, so I think that is not really necessary. 

`jaegertracing/all-in-one:latest` -> I don't mind be pointing to the latest version, but I think that is more stable to be pointing to the latest tagged version or just use the default one that comes from `@JaegerContainer`. I mean there is any motivation to use the latest jaeger version? 

If finally, you need one specific version please don't hardcore the version in the test, instead write down [in the pom](https://github.com/quarkus-qe/quarkus-test-suite/blob/main/pom.xml#L196)

Reply:
GRPC_COLLECTOR_PORT is needed because the default collector port is REST.
No reason for latest really, fallback on default